test/test.js

```diff
@@ -6,13 +6,16 @@ var figures = require('figures');
 var test = require('tape');
 var Runner = require('../lib/runner');
 var ava = require('../lib/test');
+var node = require('which').sync('node');
+var cli = path.join(__dirname, '../cli.js');
 
 function execCli(args, cb) {
 	if (!Array.isArray(args)) {
 		args = [args];
 	}
+	args.unshift(cli);
 
-	childProcess.execFile('../cli.js', args, {cwd: __dirname}, cb);
+	childProcess.execFile(node, args, {cwd: __dirname}, cb);
```

Comment:
You can just use `childProcess.execFile(process.execPath, args, {cwd: __dirname}, cb);` instead of `which`.